.github/labels.yml

```diff
@@ -31,3 +31,8 @@ docs ✍️:
   - "*.md"
 linter 🏡:
   - "test/**"
+infra :building_construction::
+  - "/*"
```

Comment:
This is going to catch docs changes (e.g., changes to `README.md` or `GOVERNANCE.md`) and some stuff that I'm not sure we even have a label for (e.g., `index.js`). Enumerating the specific files here might be a better way to go

Reply:
Actually, `index.js` and (`types.d.ts`) use the <kbd>[infra :building_construction:](https://github.com/mdn/browser-compat-data/labels/infra%20%3Abuilding_construction%3A)</kbd> label: https://github.com/mdn/browser-compat-data/pull/2338 https://github.com/mdn/browser-compat-data/pull/2308 https://github.com/mdn/browser-compat-data/pull/3739

Same with `package.json`: https://github.com/mdn/browser-compat-data/pull/3681

Reply:
Ah, nice finds! Thank you for clearing that up.